### ISSUE TITLE
Set YTD & MTD

### ISSUE THREAD

**Issue by dajjan** [RobWunderlich/Qlikview-Components#68]
Hi, Im not sure if this is a feature or a issue, but set operator YTD and MTD doesn´t show values in current date at the moment. For example we have a Sales dashboard and our sales-people really like to see there ordervalues in near real time.

At the moment max($(OrderRegvSetYTD) OrderRegDate) shows "2017-10-09"

**Comment by dajjan** [comment]
Or maybe add CurrentFullYear & CurrentFullMonth :)

**Comment by dajjan** [comment]
![skarmklipp](https://user-images.githubusercontent.com/22852231/31407829-a3c497ee-ae06-11e7-8918-c3b267996e82.PNG)

**Comment by dajjan** [comment]
Sorry to bother you, all of it sudden it started to work as it should. First the OrderReg_Qvc.Calendar_DateIsFuture stop showing me the "1" on 2017-10-10. And after a few reloads the set analysis included current date. Strange thing, I was thinking that there is someting wrong with my time but this morning it still works..

Maybe there is some varible that needs to be cleared?. I tried to find the issue in Qvc.qvs but the only thing I was able to find was that it depends on the "_f_DateIsFuture" flag.

Im not sure if this is something wrong but I also found:
SET __qvctemp.vToday=;
shouldn´t there be only on leading underscore?

For info we also need to include these lines in our script after latest release, if not paths doesen´t update.

set vG.BasePath=;
set vG.SharedBasePath=;

Regards David

**Comment by RobWunderlich** [comment]
Yes, the SET should only have one underscore. I'll correct that, but it's just cleanup and shouldn't cause the problem you described.

The current date is established in the load script.  Could it be that the script ran before midnight and then you were analyzing on the following day?

**Comment by dajjan** [comment]
Hi Rob, still having issues around this. See below snapshot. I'm not sure if this came in the QlikSense september release or if this is a bug in the QVC-script.

![image](https://user-images.githubusercontent.com/22852231/32601848-542b8aa8-c544-11e7-8f3d-2dd61b060e44.png)

**Comment by dajjan** [comment]
This is what the MTD set analysis dispays if prited in a textbox:

![image](https://user-images.githubusercontent.com/22852231/32602352-1a4bc332-c546-11e7-91e5-8187263fc13c.png)

**Comment by dajjan** [comment]
I found out that the vSetMTD shows different values depending och calandar.

This is the script generating my calendars:
CALL Qvc.CalendarFromField('Order.Date', 'OrderCalendar', 'Order.');
CALL Qvc.CalendarFromField('Service.Date', 'ServiceCalendar', 'Service.');
CALL Qvc.CalendarFromField('TerminationNotice.Date', 'TerminationNoticeCalendar', 'TerminationNotice.');
CALL Qvc.CalendarFromField('ContractExpiry.Date', 'ContractExpiryCalendar', 'ContractExpiry.');

![image](https://user-images.githubusercontent.com/22852231/32605102-6f6cd694-c550-11e7-8183-54a446c3a82a.png)

**Comment by dajjan** [comment]
This is with QVC 11.1. This is correct but the _isFuture function is not in play.. 

![image](https://user-images.githubusercontent.com/22852231/32605574-33327fc4-c552-11e7-8876-a5b1c83e0e3e.png)

**Comment by RobWunderlich** [comment]
Is Order.Date a derived field?

**Comment by dajjan** [comment]
No its not.

**Comment by dajjan** [comment]
The thing is that QVC 11.1 work just fine and I can set max calendar date to today() to achieve MTD and YTD som not show later values than Today. As you see the TerminationNotice.Date also showing different results depending on QVC version.
